samples/simple/webidl.json

```diff
@@ -0,0 +1,7 @@
+{
+    "idlFile": "simple.idl",
+    "nativeLib": "lib_simple",
+    "sourceFiles": ["point.cpp", "context.cpp"],
```

Comment:
Maybe we should add a `sourcesDir` option instead of `sourceFiles`. That is how I did the Kha integration. It is really difficult if you have a library like Bullet and you have to list each .cpp file manually.

Reply:
Yeah that's the intent but I didn't want to do all that in one PR... The next step would be to create a sourceDir...